src/components/Files/LinkToSong/LinkToSong.component.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import SongIcon from '../../SongIcon';
+import HeartFavorite from '../../HeartFavorite';
+import ContextMenu from '../../ContextMenu';
+
+const LinkToSongComponent = ({
+  fileName, isPlaying, selectSong, path,
+}) => {
+  const isPlayingClassName = isPlaying ? 'is-playing' : '';
+
+  return (
+    <div
+      onClick={selectSong}
+      role="button"
+      tabIndex={-1}
+      onKeyDown={selectSong}
```

Comment:
This event causes noise to me. This convinces me even more about having an actual button instead.